.appveyor.yml

```
@@ -87,9 +89,13 @@ matrix:
     - PIP: master
 
 install:
-    - C:\Python36\python -m pip install tox
+    - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
```

```suggestion
    - "SET PATH=%PYTHON%\\Scripts;%PATH%"
```
Shouldn't that be enough?  (but just a guess)

---

The `python` binary in `C:\Python36\` folder. That's why `%PYTHON%` was added.

---

Ok.